versioned/tiered/tiered-impl/src/main/java/org/projectnessie/versioned/impl/InternalL3.java

```diff
@@ -36,27 +38,27 @@
 
   private static final long HASH_SEED = 4604180344422375655L;
 
-  private final TreeMap<InternalKey, PositionDelta> map;
+  private final TreeMap<InternalKey, Map.Entry<PositionDelta, Integer>> map;
```

Comment:
i think this should change to a small container type rather than this generic pair